.github/workflows/basic.yml

```diff
@@ -20,10 +20,22 @@ jobs:
       matrix:
         os: [ubuntu-18.04,macos-10.15]
         type: [basic]
+        java: [8,11]
+        exclude:
```

Comment:
why excluding mac and java11?

Reply:
That was something decided with @nalinigans, though I could try it to see if it works.

Reply:
I've confirmed it works with java 11 on mac, the recent merge contains the mac 11 job.